python/lsst/ts/watcher/threshold_handler.py

```diff
@@ -0,0 +1,336 @@
+# This file is part of ts_watcher.
+#
+# Developed for Vera C. Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["ThresholdHandler"]
+
+import functools
+import math
+
+from lsst.ts.idl.enums.Watcher import AlarmSeverity
+
+from .base_rule import NoneNoReason
+
+
+# Levels must be separated by hysteresis * hysteresis_grow_factor,
```

Comment:
`hysteresis_grow_factor` -> `HYSTERSIS_GROW_FACTOR`

Reply:
I also fixed the spelling of HYSTERESIS (I tend to mis-type and omit the E before the S).